README.md

```diff
@@ -33,7 +34,7 @@ npm install
 To run it using an in-memory database (the simplest option), run:
 
 ``` sh
-LEDGER_DB_SYNC=1 LEDGER_DB_URI=sqlite://:memory: npm start
+LEDGER_ADMIN_PASS=mypassword LEDGER_DB_SYNC=1 LEDGER_DB_URI=sqlite://:memory: npm start
```

Comment:
By default, there is no admin password, which, in my experience, makes it impossible to authenticate as an admin if you run the ledger using the command here as written previously.

I'm glad to be overruled if that's my misunderstanding.